react/Installments.tsx

```diff
@@ -0,0 +1,105 @@
+import React from 'react'
+import { Installment, InstallmentOption } from 'vtex.checkout-graphql'
+import { useIntl, defineMessages } from 'react-intl'
+import { useFormattedPrice } from 'vtex.formatted-price'
+import { ListGroup, GroupOption } from 'vtex.checkout-components'
+import { useOrderPayment } from 'vtex.order-payment/OrderPayment'
+
+import CardSummary from './CardSummary'
+import Header from './components/Header'
+import { PaymentType } from './enums/PaymentEnums'
+
+const messages = defineMessages({
+  installmentValue: {
+    id: 'store/checkout-payment.installmentValue',
+  },
+  interestFree: {
+    id: 'store/checkout-payment.interestFree',
+  },
+  paymentOptionLabel: {
+    id: 'store/checkout-payment.paymentOptionLabel',
+  },
+})
+
+const InstallmentItem: React.FC<{
+  installment: Installment
+}> = ({ installment }) => {
+  const intl = useIntl()
+
+  const formattedPrice = useFormattedPrice(installment!.value! / 100)
+
+  const installmentValue = intl.formatMessage(messages.installmentValue, {
+    installments: installment.count,
+    value: formattedPrice,
+  })
+
+  const interestRate =
+    installment.hasInterestRate || installment.count === 1
+      ? ''
```

Comment:
No risk here to use an empty string? What if you'd use `0`?

Reply:
don't think so, because this is the message that will be rendered. I can rename this variable to `interestRateMessage` so its clearer